code/track.ss

```diff
@@ -5,17 +5,46 @@
       (delete-file config.json))
     (with-output-to-file config.json
       (lambda ()
-	(json-write (process-config))))))
+        (json-write 'pretty (process-config))))))
 
+;; Top level helper for make-config
 (define (process-config)
   (map (lambda (x)
          (if (not (eq? (car x) 'exercises))
              x
-             `(exercises . ,(remp (lambda (exercise)
-                                    (memq 'wip (map car exercise)))
-                                  (cdr x)))))
+             (cons 'exercises
+                   (exercises->snake-case
+                    (remp (lambda (exercise)
+                            (memq 'wip (map car exercise)))
+                          (cdr x))))))
        track-config))
 
+;; Top level helper for process config
```

Comment:
Tried to organize these helpers with comments.  I originally most of the helpers packed into `exercises->snake-case`, but it is much easier to read this way.  This makes me feel nervous about how much global namespace polution is going on though, so I would love if these could go into a library.